app/shared/date-picker/_date-picker.scss

```diff
@@ -87,7 +91,7 @@
     background-color: $light-gray;
     border-radius: 0;
     font-size: 1.6rem;
-  
+
     &:focus {
       outline: none;
     }
```

Comment:
And maybe here we can bring it back?

Reply:
It's just linter work, i think they better with space..

Reply:
Yeah, I meant the outline was turned off within a previous PR, maybe we can bring it back :)

Reply:
Consider it as done